test/scenarios/jmx/docker-compose-kafka3.yml

```
@@ -0,0 +1,6 @@
+version: '2'
+services:
+  kafka:
+    environment:
+      KAFKA_LISTENERS: PLAINTEXT://:9092
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka_1:9092
```

This should be the hostname (service) of the container

```suggestion
      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:9092
```

---

nice catch, fixed (did not know about commit suggestion.. I do now)